stored_requests/backends/http_fetcher/fetcher.go

```diff
@@ -56,9 +57,10 @@ func NewFetcher(client *http.Client, endpoint string) *HttpFetcher {
 }
 
 type HttpFetcher struct {
-	client   *http.Client
-	Endpoint string
-	hasQuery bool
+	client     *http.Client
+	Endpoint   string
+	hasQuery   bool
+	Categories map[string]map[string]file_fetcher.Category
```

Comment:
We should move the file_fetcher.Category type into a common module so that http_fetcher doesn't have to depend on file_fetcher.

Reply:
Agree. Done